src/Microsoft.TestPlatform.AdapterUtilities/Extensions/ReflectionExtensions.MethodBase.cs

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Extensions
+namespace Microsoft.TestPlatform.AdapterUtilities.Extensions
 {
     using System;
     using System.Reflection;
 
-    public static partial class ReflectionExtensions
+    internal static partial class ReflectionExtensions
```

Comment:
Have we checked the VS repo to ensure this does not have any references that could break?

Reply:
The class itself was introduced as part of FQN, never used anywhere since it didn't ship.